components/dashboard/src/prebuilds/InstallGitHubApp.tsx

```diff
@@ -16,13 +16,6 @@ async function registerApp(installationId: string, setModal: (modal: 'done' | st
     try {
         await getGitpodService().server.registerGithubApp(installationId);
```

Comment:
Nit: If possible, every `window.open()` call should always happen in the exact same event loop cycle as the `click` event.

Otherwise the `window.open()` will most likely be blocked by the browser's pop-up blocker. This means, no `await`, no callbacks, etc.

Reply:
Can we change the order?